docs/simulariumio.data_objects.rst

```diff
@@ -36,6 +36,14 @@ simulariumio.data\_objects.display\_data module
    :undoc-members:
    :show-inheritance:
 
+simulariumio.data\_objects.file\_data module
+--------------------------------------------
+
+.. automodule:: simulariumio.data_objects.file_data
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
```

Comment:
yes from `make docs`

Reply:
Yeah I wouldn't think it would cause any issues, and maybe it's hard to have it git-ignored here, and still published to the gh-pages branch, but just for the sake of cleaner/smaller PRs, it might be nice to look into